src/daemon/jsonrpc/api/mod.rs

```diff
@@ -208,6 +208,7 @@ pub trait RpcApi {
         &self,
         meta: Self::Metadata,
         spend_txid: Txid,
+        priority: bool,
```

Comment:
Any reason for not making it an optional parameter? (no opinion, just asking)

Also needs a API.md update

Reply:
Well `priority` is just two states: either the tx has priority or it doesn't. It seemed weird to represent this in the API with a three states scenario, where both `None` and `Some(false)` mean "no priority" and `Some(true)` means "with priority".

On the other side, the GUI needs an update if this is not optional (at first, since we don't have the UI for asking for automatic CPFP, we would just send `priority: false` all the time)

Reply:
Actually, thinking more about it, it makes sense to just have an Option and default it to false. Will update!